ocs_ci/framework/conf/default_config.yaml

```diff
@@ -52,6 +58,9 @@ REPORTING:
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
   cluster_name: null  # will be changed in ocscilib plugin
+  storage_cluster_name: 'rook-ceph'
+  storage_device_sets_name: "storageDeviceSets"
+  device_size: '100Gi'
```

Comment:
Yep, those values should be overwritten by custom cluster config. And there is gonna be more of those so it will be hell to maintain every configuration of cluster as CLI args. Anyway, I set now as default 100Gigs but for production runs we will have custom config with 1 Tera as this should be default for customers ENVs.

Reply:
Resolving this conversation